tests/TestCase/Error/ExceptionRendererTest.php

```diff
@@ -702,61 +702,6 @@ public function testExceptionNameMangling()
         $this->assertStringContainsString('widget thing is missing', $result);
     }
 
-    /**
-     * Test exceptions being raised when helpers are missing.
-     *
-     * @return void
-     */
-    public function testMissingRenderSafe()
-    {
-        $exception = new MissingHelperException(['class' => 'Fail']);
-        $ExceptionRenderer = new MyCustomExceptionRenderer($exception);
-
-        /** @var \Cake\Controller\Controller|\PHPUnit\Framework\MockObject\MockObject $controller */
-        $controller = $this->getMockBuilder('Cake\Controller\Controller')
-            ->onlyMethods(['render'])
-            ->getMock();
-        $controller->viewBuilder()->setHelpers(['Fail', 'Boom']);
-        $controller->setRequest(new ServerRequest());
-        $controller->expects($this->once())
-            ->method('render')
-            ->with('missingHelper')
-            ->will($this->throwException($exception));
-
-        $ExceptionRenderer->setController($controller);
-
-        $response = $ExceptionRenderer->render();
-        $helpers = $controller->viewBuilder()->getHelpers();
-        sort($helpers);
-        $this->assertEquals([], $helpers);
-        $this->assertStringContainsString('Helper class Fail', (string)$response->getBody());
```

Comment:
Does this error no longer get rendered?

Reply:
All non-template errors are treated as internal errors now and trigger the fallback case in ErrorHandlerMiddleware.